src/Reveal.Sdk.Dom.Tests/Data/DataSourceItems/AmazonAthenaDataSourceItemFixture.cs

```diff
@@ -0,0 +1,97 @@
+using Reveal.Sdk.Dom.Data;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Data.DataSourceItems
+{
+    public class AmazonAthenaDataSourceItemFixture
+    {
+        [Fact]
+        public void Constructor_SetsTitleAndDataSource_ValidDataSource()
+        {
+            // Arrange
+            string title = "Test Item";
+            var dataSource = new AmazonAthenaDataSource();
+            var item = new AmazonAthenaDataSourceItem(title, dataSource);
+
+            // Act
+            var actualTitle = item.Title;
+            var actualDataSource = item.DataSource;
+
+            // Assert
+            Assert.Equal(title, actualTitle);
+            Assert.Equal(dataSource, actualDataSource);
+        }
+
+        [Fact]
+        public void Constructor_SetsTitleAndDataSource_NullDataSource()
+        {
+            // Arrange
+            string title = "Test Item";
+            DataSource dataSource = null;
+            var item = new AmazonAthenaDataSourceItem(title, dataSource);
+
+            // Act
+            var actualTitle = item.Title;
+            var actualDataSource = item.DataSource;
+
+            // Assert
+            Assert.Equal(title, actualTitle);
+        }
+
+        [Fact]
+        public void Constructor_SetsTitle_EmptyTitle()
+        {
+            // Arrange
+            string title = string.Empty;
+            var dataSource = new AmazonAthenaDataSource();
+            var item = new AmazonAthenaDataSourceItem(title, dataSource);
+
+            // Act
+            var actualTitle = item.Title;
+
+            // Assert
+            Assert.Equal(title, actualTitle);
+        }
+
+        [Fact]
+        public void HasTabularData_ReturnsTrue_Always()
```

Comment:
Remove, only test new fields that this class add, if any, not the parent class's fields.